src/groups/mqb/mqbblp/mqbblp_queuehandle.cpp

```diff
@@ -499,7 +498,7 @@ void QueueHandle::deliverMessageImpl(
                      subQueueInfos.size() <= d_subscriptions.size());
 
     d_domainStats_p->onEvent<mqbstat::QueueStatsDomain::EventType::e_PUSH>(
-        msgSize);
+        attributes.appDataLen());
```

Comment:
Might as well add precondition `appDataLen() > 0`, because we don't allow to post a message with 0 app data length.